File: Pruba Tecnica Cordoba/WebApiUsuarios/WebApiUsuarios/Models/Usuario.cs
namespace WebApiUsuarios.Models
{
    public class Usuario
    {
        public string? Nombre { get; set; }
        public string? Apellido { get; set; }
        public string? Cedula { get; set; }
        public string? CorreoElectronico { get; set; }
        public string? Contrasena { get; set; }
        public int Puntaje { get; set; }
    }
}

File: Pruba Tecnica Cordoba/Usuarios/src/app/shared/models/Usuario.ts
export interface Usuario {
    nombre: string;
    apellido: string;
    cedula: string;
    correoElectronico: string;
    contrasena: string;
    puntaje: number;
}

export interface Login {
    correoElectronico: string;
    Contrasena: string;
}

export interface GetUsuario{
    idUsuario: number;
    nombre: string;
    apellido: string;
    cedula: string;
    correoElectronico: string;
    fechaHoraUIngreso: string;
    contrasena: string;
    puntaje: number;
    idClasificacionUsuario: number;
    nombreClasificacion: string;
    fechaCreacion: string;
}

export interface UpdateUsuario{
    nombre: string;
    apellido: string;
    cedula: string;
    correoElectronico: string;
    contrasena: string;
    puntaje: number;
    idUsuario: number;
}